libenkf/src/ert_run_context.c

```diff
@@ -86,7 +90,15 @@ stringlist_type * ert_run_context_alloc_runpath_list(const bool_vector_type * ia
 }
 
 
-static ert_run_context_type * ert_run_context_alloc(bool_vector_type * iactive , run_mode_type run_mode , enkf_fs_type * init_fs , enkf_fs_type * result_fs , enkf_fs_type * update_target_fs , int iter) {
+char * ert_run_context_alloc_run_id( ) {
+  int year,month,day,hour,min,sec;
+  time_t now = time( NULL );
+  unsigned int random = util_dev_urandom_seed( );
+  util_set_datetime_values_utc( now , &sec, &min, &hour, &day, &month, &year);
+  return util_alloc_sprintf("%d:%d:%4d-%0d-%02d-%02d-%02d-%02d:%ud" , getpid() , getuid(), year , month , day , hour , min , sec, random);
```

Comment:
> [....] I would like to have the uid, ohh it is right there in the ID, let me just extract it

I thought of that too - and I agree with you. Will change it.